### PR TITLE
Feature/417 social sharing

### PR DESCRIPTION
**Pull request by nicklee**
https://github.com/mozilla/donate-wagtail/issues/417

### CODE REVIEW

File: donate/templates/payment/thank_you.html [hunk 1]
```diff
@@ -18,30 +18,41 @@
...
                     <svg class="button__icon button__icon--facebook" width="16" height="16">
                         <use xlink:href="#facebook"></use>
                     </svg>
                     <div class="button__label">{% trans "Facebook" %}</div>
                 </a>
-                <a class="button button--outline button--icon" href="https://twitter.com/mozilla">
+                {# see https://dev.twitter.com/web/tweet-button/web-intent #}
+                <a class="button button--outline button--icon" href="https://twitter.com/intent/tweet?text=I donated to @mozilla today because I %23lovetheweb. Join me and help fight for a better and healthier internet, for all.&amp;url=http://donate.mozilla.org&amp;via=mozilla" title="Share on Twitter">
```
Comment (TheoChevalier) [inline donate/templates/payment/thank_you.html]: @nicklee Note that this needs to be localized *then* urlencoded (and same for FB and email)

Comment (nicklee) [reply donate/templates/payment/thank_you.html]: @TheoChevalier Updated with localisation and urlencoding

Comment (TheoChevalier) [reply donate/templates/payment/thank_you.html]: Cheers!

**Comment by TheoChevalier** [comment]
@nicklee Do you mind if we merge this one now and potentially update the FB stuff in a follow up? This is my last blocker to start localization in Pontoon

**Comment by nicklee** [comment]
@TheoChevalier that's fine with me, we just need to make sure we address the facebook issue before launch!

**Pull request merged**